XTS/WS-C/dev/src/com/arjuna/webservices11/util/ServiceAction.java

```diff
@@ -28,10 +30,18 @@ public static <T extends Service> ServiceAction<T> getInstance(final Class<T> se
     @Override
     public T run() {
         try {
-            return serviceClass.newInstance();
+            return serviceClass.getDeclaredConstructor().newInstance();
         } catch (final InstantiationException | IllegalAccessException e) {
             WSCLogger.i18NLogger.warn_cannot_create_service_instance(serviceClass, e);
             throw new RuntimeException(e);
         }
-    }
+		catch (InvocationTargetException e) {
+            WSCLogger.i18NLogger.warn_cannot_create_service_instance(serviceClass, e);
+            throw new RuntimeException( e );
+		}
+		catch (NoSuchMethodException e) {
+            WSCLogger.i18NLogger.warn_cannot_create_service_instance(serviceClass, e);
+            throw new RuntimeException( e );
+		}
+	}
```

Comment:
Please can I request spaces instead of tabs here for the `catch ...` and `}` lines?

Reply:
Could we group all exceptions here as well? WDYT @tomjenkinson?

Reply:
+1 @jmfinelli, great point. I think we could add ` | InvocationTargetException | NoSuchMethodException` to the existing catch clause (https://github.com/jbosstm/narayana/pull/2329/files/1f96757888138be57564f95bf0a616eb33ecacfd#diff-e487a2c1d88a93d1befdff0434ff66d6f593275e0ee1db9a64672e3c6d7366d4R34)